R/buoyancy.freq.R

```diff
@@ -2,6 +2,40 @@
 # Buoyancy Frequency

```

Comment:
Now that there is full Roxygen documentation on these functions, you can drop this opening text to make the files cleaner. It is not longer necessary.

Reply:
Addressed here: https://github.com/boshek/rLakeAnalyzer/commit/2cb07adb456c7544e0f4a6988f7e379270e2285f